src/integration-test/system-integration-tests.ts

```diff
@@ -4,879 +4,305 @@
+          this.timeout(1000 * 20);
+          const teamId = await client.getTeamId();
+          for (const notificationType of Object.keys(config.notifications)) {
+            const attributes = config.notifications[notificationType as NotificationType];
+            if (attributes) {
+              const _attributes = {
+                teamId,
+                projectId: null,
+                ...attributes,
+              };
+              const response = await client.configureNotification(_attributes);
+              notificationIds.push(response.id);
+            }
+          }
+        });
+
+        it('should be able to configure project scoped notifications', async function () {
+          this.timeout(1000 * 20);
+          const teamId = null;
+          const projectId = client.lastProject!.id;
+          for (const notificationType of Object.keys(config.notifications)) {
+            const attributes = config.notifications[notificationType as NotificationType];
+            if (attributes) {
+              const _attributes = {
+                teamId,
+                projectId,
+                ...attributes,
+              };
+              const response = await client.configureNotification(_attributes);
+              notificationIds.push(response.id);
+            }
+          }
+        });
       });
-      expect(ret.status).to.equal(200);
-      log('Notification configuration deleted');
-    });
 
-    it('should be able to delete configured Slack notifications', async function () {
-      this.timeout(1000 * 10);
-      if (!slackWebhookUrl) {
-        log('No slackWebhookUrl defined. Skipping deletion of notification configuration');
-        return;
-      }
-      const ret = await userFetchWithRetry(`${charles}/api/notifications/${slackNotificationId}`, {
-        method: 'DELETE',
+      describe('comments', () => {
+        let comment: undefined | JsonApiEntity;
+        it('should be able to add comment for deployment', async function () {
+          this.timeout(1000 * 10);
+          const message = 'integration test message';
+          const email = 'user@integration.com';
+          const name = 'Charles Minard';
+          comment = await client.addComment(deployment!.id, message, name, email);
+          expect(comment.attributes.message).to.equal(message);
+          expect(comment.id).to.exist;
+        });
+
+        it.skip('should be able to fetch comments for deployment', async function () {
+          this.timeout(1000 * 10 * 6);
+          await sleep(20000);
+          const comments = await client.getComments(deployment!.id);
+          expect(comments.length).to.equal(1);
+          expect(comments[0].attributes.message).to.equal(comment!.attributes.message);
+        });
+
+        it('should be able to delete comment for deployment', async function () {
+          this.timeout(1000 * 10);
+          await client.deleteComment(comment!.id);
```

Comment:
`CharlesClient` always throws if the response status code is not what was expected. Payload wise there's nothing to check.